### PR TITLE
ntp: keyboard support for stats

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208590466011386/f

## Description

- [x] back porting accessible keyboard support (from favorites branch)
- [x] co-locate test files (playwright has no issue finding them)

## Testing Steps

- visit https://deploy-preview-1149--content-scope-scripts.netlify.app/build/pages/new-tab/
- ensure you can tab from the address bar, onto the control (see the video)
- ensure you can open/close the stats with space bar

https://github.com/user-attachments/assets/6ec58ae0-1313-447f-aa3b-6567266bc11c

## Checklist

*Please tick all that apply:*

- [ ] I have tested this change locally
- [ ] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [ ] I have added automated tests that cover this change
- [ ] I have ensured the change is gated by config
- [ ] This change was covered by a ship review
- [ ] This change was covered by a tech design
- [ ] Any dependent config has been merged

### CODE REVIEW

**Comment by shakyShane** [comment]
* **#1149** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1149?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @shakyShane and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: special-pages/pages/new-tab/app/components/ShowHideButton.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <button
+            {...buttonAttrs}
+            class={styles.button}
+            aria-label={text}
+            onClick={onClick}
+        >
+            <ChevronButton/>
+        </button>
+    )
+}
```
Comment (shakyShane) [inline special-pages/pages/new-tab/app/components/ShowHideButton.jsx:24]: back-porting this re-use from the favorites PR - where it gets re-used

File: special-pages/pages/new-tab/app/privacy-stats/PrivacyStats.js [hunk 2]
```diff
@@ -26,10 +24,6 @@ import { viewTransition } from '../utils.js'
  * @param {Animation['kind']} [props.animation] - optionally configure animations
  */
 export function PrivacyStats ({ expansion, data, toggle, animation = 'auto-animate' }) {
-    if (animation === 'auto-animate') {
-        return <WithAutoAnimate data={data} expansion={expansion} toggle={toggle} />
-    }
-
```
Comment (shakyShane) [inline special-pages/pages/new-tab/app/privacy-stats/PrivacyStats.js:32]: auto-animate is problematic for a few reasons, dropping it

File: special-pages/pages/new-tab/src/locales/en/newtab.json [hunk 3]
```diff
@@ -39,6 +39,18 @@
...
+    "note": "The aria-label text for a toggle button that hides the detailed activity feed"
+  },
+  "favorites_show_less": {
+    "title": "Show less",
+    "note": ""
+  },
+  "favorites_show_more": {
+    "title": "Show more ({count} remaining)",
+    "note": ""
   }
```
Comment (shakyShane) [inline special-pages/pages/new-tab/src/locales/en/newtab.json:55]: from the next branch, no harm being here now.

**Review by mgurgel — APPROVED** [review]
Code looks good. Tabbing works on Chrome and Safari. How do I get it to work on DDG though? I don’t see an option similar to the Safari one 

image

**Comment by shakyShane** [comment]
@mgurgel working in chromium right now is enough to move forward in windows - we can address any other platforms later :)

**Pull request merged**